class CustomException(Exception):
    pass

# def div(a,b):
#     try:
#         a//b
#     except ZeroDivisionError as ex:
#         try:
#             raise CustomException()
        
         
            
